server/src/main/java/com/exacaster/lighter/application/sessions/PermanentSession.java

```diff
@@ -0,0 +1,26 @@
+package com.exacaster.lighter.application.sessions;
+
+import com.exacaster.lighter.application.Application;
+import com.exacaster.lighter.application.SubmitParams;
+
+public class PermanentSession {
+    private final Application application;
+    private final boolean isDeleted;
```

Comment:
I think it is fine to add "isDeleted" directly to the `Application`, in that case `PermanentSession` class would not probably be needed. Or are there any more reasons for it to exist?

Reply:
Ok, done